CHANGES.rst

```diff
@@ -5,6 +5,7 @@ extract_1d
 ----------
 
 - Fix error in variance propagation calculation [#6899]
+- Fix error in setting DQ flags for s3d data when on data is in extraction aperture [#6909]
```

Comment:
Minor nit, but this could use a little rewording to make it more obvious exactly where/when DQ flags are set and why. I suggest "Set DO_NOT_USE flag in extracted spectrum when the IFU extraction aperture has no valid data"